Update use of CA instead old features (#11519)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md

If your organization has more granular sign-in security needs, Conditional Access policies can offer you more control. Conditional Access lets you create and define policies that react to sign in events and request additional actions before a user is granted access to an application or service.

> [!IMPORTANT]

> > Turn off Security defaults before you enable Conditional Access policies. It's advised to disable Per-User MFA also along with Conditional Access deployment since it offers a more granular security approach to access cloud resources.

Reviewer: ```suggestion
> Turn off Security defaults before you enable Conditional Access policies. We recommend disabling per-user MFA along with Conditional Access deployment since it offers a more granular security approach to accessing cloud resources.
```
